test/cli/remote-file/download-file-disk-without-decompress/initial-state.json

```diff
@@ -0,0 +1,18 @@
+{
+  "branches": [
+    {
+      "branch": {
+        "name": "Main",
+        "description": "",
+        "isDefault": true
+      },
+      "configs": []
+    }
+  ],
+  "files": [
+    {
+      "name": "test1",
+      "content": "H4sIAAAAAAAAA0vOzzHUSc7PMeIqSwSygASYZQximQAAVQ0b1R0AAAA="
```

Comment:
I am not able to decompress this, but we would need binary writer instead of string one.